tests/scripts/arm32_ci_script.sh

```diff
@@ -297,13 +357,48 @@ function run_tests {
 EOF
 }
 
+function run_tests_using_docker {
+    __currentWorkingDirectory=`pwd`
+
+    # Configure docker
+    if [ "$__buildArch" == "arm" ]; then
+        case $__linuxCodeName in
+        trusty)
+            __dockerImage=" chcosta/dotnetcore:ubuntu1404_cross_prereqs_v1"
```

Comment:
I will update this too after #9853 is merged, becasue I can't test for arm with it in CI yet.